strict/strict.cabal

```diff
@@ -82,6 +82,7 @@ library
     , bytestring   >= 0.9.2.1 && < 0.11
     , deepseq      >= 1.3.0.0 && < 1.5
     , hashable     >= 1.2.7.0 && < 1.4
+    , serialise    >= 0.2.0.0 && < 0.3
```

Comment:
This is unacceptable. What would add nontrivial `serialise` dependency to e.g. `aeson`.

Reply:
@phadej This type of language in English has some hostile connotations. Can you explain why `serialise` is less trivial than say, `binary` which this already depends on?

Reply:
`binary` is bundled with GHC.

Reply:
To elaborate. I convinced maintainers of `aeson` and `lens` to depend on `strict` with a promise that `strict` is light package, and doesn't bring any extra dependencies. I cannot break that promise, or would need to rip off `strict` out of `lens` and `aeson` and we'll be back to where we started.

If you need `serialise` instances, make `serialise-strict` package or (better) try to make `serialise` depend on `strict`.

Reply:
Also there is `cereal`, `store`.. just too many encoders/decoders do depend upon / provide instances for.